kolibri/core/assets/src/views/k-navbar/index.vue

```diff
@@ -51,6 +51,7 @@
     mixins: [responsiveElement],
     data: () => ({
       enoughSpace: true,
+      rtl: this.isRtl || false,
```

Comment:
why is this data and not a computed prop?

also, why is it necessary at all - add a comment maybe?

Reply:
Yeah - this should never be undefined (if it is, let's fix that instead).

Reply:
rtl was undefined in the style guide. I'll just revert this, don't want to pollute this component because of the broken style guide. :call_me_hand:

Reply:
Ahh, we can definitely fix that independently also! (styleguide works quite nicely in Arabic)